dsc_lib/src/discovery/command_discovery.rs

```diff
@@ -6,25 +6,32 @@ use crate::discovery::convert_wildcard_to_regex;
 use crate::dscresources::dscresource::{Capability, DscResource, ImplementedAs};
 use crate::dscresources::resource_manifest::{import_manifest, validate_semver, Kind, ResourceManifest};
 use crate::dscresources::command_resource::invoke_command;
-use crate::dscresources::command_resource::log_resource_traces;
 use crate::dscerror::DscError;
 use indicatif::ProgressStyle;
 use regex::RegexBuilder;
+use semver::Version;
 use std::collections::{BTreeMap, HashSet};
 use std::env;
 use std::ffi::OsStr;
 use std::fs::File;
 use std::io::BufReader;
 use std::path::{Path, PathBuf};
-use tracing::{debug, error, trace, warn, warn_span, Span};
+use tracing::{debug, info, trace, warn, warn_span};
 use tracing_indicatif::span_ext::IndicatifSpanExt;
 
 pub struct CommandDiscovery {
+    // use BTreeMap so that the results are sorted
```

Comment:
please extend the comment to something like :
... so that results are sorted by reousrce type name and vector values - are found resource versions of this type in order of decreasing of semversion.

Reply:
fixed